spec/PhpSpec/Formatter/Presenter/StringPresenterSpec.php

```diff
@@ -26,7 +26,7 @@ function it_presents_long_string_in_quotes_but_trimmed()
 
     function it_presents_only_first_line_of_multiline_string()
     {
-        $this->presentValue("some\nmultiline\nvalue")->shouldReturn('"some..."');
+        $this->presentValue("some".PHP_EOL."multiline".PHP_EOL."value")->shouldReturn('"some..."');
```

Comment:
changing this is wrong. We want to test explicitly what happens when the string contains a ``\n``, even on Windows